README.md

```diff
@@ -1,75 +1,59 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Capstone project: Entrepreneurs Forum Summit landing page
 
 > Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
+- Bootstrap
+- Linters (CSS and Javascript)
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://tessyrich.github.io/Capstone-Project-Main/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+To get a local copy up, CD into the project folder and run:
+"""
+npm install
+"""
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
 
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
 
 ### Deployment
+Deployed using GitHub
 
 
+👤 **Author**
 
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
+- GitHub: [@githubhandle](https://github.com/Tessyrich)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] Please replace these placeholders with your profile handles. Currently, this is not looking professional. Also if you don't have twitter or any other listed, you may rather remove it than keep it there because it can creat a bad impletion when one clicks on it, and then nothing happens. I hope you understand what I am suggesting 😎